packages/calcite-components/src/components/panel/panel.e2e.ts

```diff
@@ -721,7 +721,6 @@ describe("calcite-panel", () => {
       </calcite-panel>`,
       {
         "--calcite-panel-corner-radius": {
```

Comment:
Can this test be updated to cover the internal action CSS prop overrides?

Reply:
it isn't necessary

Reply:
Other tests cover this (e.g., [inline-editable](https://github.com/Esri/calcite-design-system/pull/...), [flow-item](https://github.com/Esri/calcite-design-system/pull/...)), so we should do the same here for consistency.

Reply:
Based on https://github.com/Esri/calcite-design-system/pull/11495#discussion_r1964717901, my comment is not applicable.